include/dt-bindings/dma/stm32_dma.h

```diff
@@ -1,34 +0,0 @@
-/*
```

Comment:
This change should be moved to another commit

Reply:
moved to commit a6a6897